src/material/select/testing/select-harness.ts

```diff
@@ -7,9 +7,7 @@
  */
 
 import {HarnessPredicate, TestElement} from '@angular/cdk/testing';
-import {
-  MatFormFieldControlHarness
-} from '@angular/material-experimental/form-field/testing/control';
+import {MatFormFieldControlHarness} from '@angular/material/form-field/testing/control';
 import {SelectHarnessFilters} from './select-harness-filters';
 import {MatSelectOptionHarness, MatSelectOptionGroupHarness} from './option-harness';
```

Comment:
Note that we have a harness for an option inside `mat-select`, as well as for an option in `mat-autocomplete`. Once this PR is merged in I can remove the duplication and add single, fully-featured harness for `mat-option` in `core`.

Reply:
It would be really nice if we could get that in before final. I forgot that autocomplete was already moved out of experimental with the split mat-option harness